test/common/model/fields.spec.js

```diff
@@ -1,90 +0,0 @@
-const chai = require('chai');
```

Comment:
`src/js/common/model/fields.js` still exists, so should this test really be deleted?